text_generation/causal_lm/cpp/continuous_batching/apps/throughput_benchmark.cpp

```diff
@@ -355,6 +356,63 @@ void statisticsReporter(GenerationInfoCollector* generations_info_collector, int
     std::cout << "Exiting statistics reporter thread." << std::endl;
 }
 
+bool parse_plugin_config_json(nlohmann::json& node, ov::AnyMap& device_config_map) {
```

Comment:
General comment not connected with the PR. I have a problem with `std::atomic<bool>` while building benchmark. Could you please fix it?